### ISSUE TITLE
[ejabberdctl join_cluster] All clients get disconnected whenever a single node is shutdown.

### ISSUE THREAD

**Issue by joudinet** [processone/ejabberd#740]
Hi,

I followed the documentation to establish a cluster of ejabberd using `ejabberdctl join_cluster`.

AFAIK, I should be able to shutdown any node from the cluster and clients connected to other nodes should remain connected. This is not what happens in my case study.
I have two running nodes in my cluster:
```
# su ejabberd
$ ejabberdctl join_cluster 'ejabberd@10.43.52.114'
# ejabberdctl start
# ejabberdctl mnesia | grep db_nodes
 {db_nodes,['ejabberd@10.43.52.114','ejabberd@10.40.18.59']},
 {extra_db_nodes,[]},
 {running_db_nodes,['ejabberd@10.43.52.114','ejabberd@10.40.18.59']},
```
There is one client connected to each node, and they chat on a shared MUC resource. So far, so good.
When I shutdown `10.43.52.114`, both clients get disconnected :-\
When I reconnect to the MUC service with the client connected to `10.40.18.59`, it does reconnect but every message from the room have been discarded.
Why the client connected to the still running node gets disconnected?

**Comment by joudinet** [comment]
I did further experiments on this issue and find out that if I disconnect a node from the cluster without shutting down properly, the clients connected to other nodes are not disconnected from the MUC room. Since I'm using ejabberd 15.06, this looks like the bug fixed by a16834083858b76da98b9cbfe39c5df7de9fac1a. I close this issue until I test with ejabberd 15.07.

**Comment by mremond** [comment]
Thank you for the update !
